heartbeat/monitors/wrappers/monitorstate/esloader.go

```diff
@@ -54,9 +59,9 @@ func MakeESLoader(esc *eslegclient.Connection, indexPattern string, beatLocation
 			},
 		}
 
-		if sf.RunFrom != nil {
+		if runFromID != "" {
```

Comment:
Found a bug with this approach, As we are passing the state query only when runFrom is specified. It leads to a bug when then runFrom is set and removed later. 

1. Set `heartbeat.runfrom` to a `test-loc`, state will created on index `<monitorstate:id=test-loc-<blah>>`
2. Remove the value of `run_from`, now instead of checking the `default` location or unknown if we change the naming. We will be look at the wrong index for the monitor.